_docs/whatiscloud.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+category: Getting Started
+title: Cloud Services FAQ
+order: 1
+permalink: /clsvcfaq.html
+---
+
+# Cloud Services FAQ
+
+If you're new to the idea of working with the cloud and you want a quick and easy primer to get you up and running, you're in the right place. This page will cover what the cloud is, how it works, and why you should care.
+
+### So what's the cloud anyway?
+Quite simply, the cloud is a way to run your software on someone else's machines. In most cases, developing software for the cloud is just like developing software for running locally. This allows you to build complex online applications using the same programming languages and concepts you’re already familiar with while avoiding the hassle of provisioning and maintaining infrastructure.
+
+There are two kinds of offerings that you might see when browsing a cloud provider’s website. Briefly, these are **Virtual Machines** and **Services**.
+
+* Virtual Machines, or VMs, are exactly what they sound like. They’re a virtual environment that emulates a real computer, hosting an Operating System and any software you might want to run. For example, when using a VM as a server, you need to setup packages, configure settings, and run your server application manually, just as you would if you setup your own computer as a server. Cloud VM offerings include Amazon’s AWS EC2 and Azure’s VMs. VMs won’t be covered in detail here. For now, just know that they exist and what they’re for.
+
+* Services, on the other hand, abstract away the nitty-gritty implementation details. Services allow you to build and deploy your application without worrying about the underlying details. For example, Amazon’s AWS Lambda service allows you to deploy functions that contain regular code. AWS might run your code in a VM, or in a Container. The key thing to remember here is, *it doesn’t matter to you*. Most cloud offerings fall into the Services category, and they make developing cloud applications much easier than it might appear.
+
+### Intriguing...so what can I do in the cloud?
+Almost anything you can dream of. Cloud providers are constantly improving and adding to their offerings. A good place to start would be to take a simple application that you could run locally, such as an app that retrieves and stores information in a database and move it into the cloud. Refer to [LINK HERE] for a few concrete examples.
```

Comment:
To do: replace [LINK HERE] with link to a docs page. Feel free to add a link to an internal doc page or an external page.